clients/adapter-amqp/src/main/java/org/eclipse/hono/adapter/client/command/amqp/ProtonBasedLegacyCommandContextWrapper.java

```diff
@@ -63,15 +63,25 @@ public void release() {
         ctx.release();
     }
 
+    @Override
+    public void release(final Throwable error) {
+        Objects.requireNonNull(error);
+        ctx.release();
```

Comment:
shouldn't this invoke `ctx.release(Throwable)`?

Reply:
The deprecated `org.eclipse.hono.client.CommandContext` class doesn't have such a method.

Reply:
Ah, I see. It's the deprecated one ...